src/components/Chart_M.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
Do we need Magda's changes here?
Please keep it granular (one thing as a time), so pull request is easy to understand.